notebooks/data analysis/Pathfinder_system_other_columns.ipynb

```diff
@@ -0,0 +1,3769 @@
+     "output_type": "stream",
+     "text": [
+      "41\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in range(will.value.min(), will.value.max() + 1):\n",
+    "    if will[will.value == i].size < 1:\n",
+    "        print(i)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Saves - podsumowanie\n",
+    "mamy trzy kolumny: fortitude, reflex, will\n",
+    "\n",
+    "Wszystkie potworki mają nie puste values\n",
+    "\n",
+    "Zakres wartości:\n",
+    "* fortitude: 0-44\n",
+    "* reflex: 0-41\n",
+    "* will: 0-43\n",
+    "Z jakiegoś powodu w każdej kolumnie istnieje dokładnie jadna wartość której nie ma żaden potworek (z powyższych przediałów)\n",
+    "\n",
+    "**Wydaje mi się, że dane dość spoko do w miarę prostego dodania**\n",
+    "Czy są istotne?"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Traits"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 75,
+   "outputs": [],
+   "source": [
+    "traits = unpack_column(system, \"traits\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 78,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.frame.DataFrame'>\n",
+      "Int64Index: 1206 entries, 0 to 364\n",
+      "Data columns (total 6 columns):\n",
+      " #   Column     Non-Null Count  Dtype \n",
+      "---  ------     --------------  ----- \n",
+      " 0   languages  1206 non-null   object\n",
+      " 1   rarity     1206 non-null   object\n",
+      " 2   senses     1206 non-null   object\n",
+      " 3   size       1206 non-null   object\n",
+      " 4   value      1206 non-null   object\n",
+      " 5   attitude   718 non-null    object\n",
+      "dtypes: object(6)\n",
+      "memory usage: 66.0+ KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "traits.info()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 76,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "                                           languages    rarity  \\\n0  {'custom': '', 'selected': [], 'value': ['comm...    common   \n1        {'custom': '', 'selected': [], 'value': []}    common   \n2        {'custom': '', 'selected': [], 'value': []}    common   \n3  {'custom': '', 'selected': [], 'value': ['abys...  uncommon   \n4  {'custom': '', 'selected': [], 'value': ['cele...    common   \n\n                                              senses               size  \\\n0  {'value': 'darkvision, scent (imprecise) 60 fe...    {'value': 'lg'}   \n1  {'value': 'low-light vision, scent (imprecise)...   {'value': 'grg'}   \n2  {'value': 'low-light vision, scent (imprecise)...  {'value': 'huge'}   \n3  {'value': 'darkvision
```

Comment:
Może się przyda, bo pewnie każda subrasa czy coś tam ma jakieś właściwości. Ale oby nie miała bo tego jest dużo. I z każdą książką prawdopodobnie będzie tego więcej!

Reply:
Types są chyba nadrzędne na tym i oby to nam wystarczyło